app/Http/Controllers/visitController.php

```diff
@@ -41,6 +44,48 @@ public function index(): View
         );
     }
 
+    /**
+     * 
+     * @param string $slug
+     * @return View
+     * 
+     * This function looks for a subpage from the 'Subpages' collection in Directus
+     * first, then looks for a subpage from the 'Stubs and Pages' collection in Directus
+     * if a page in the 'Subpages' collection couldn't be found.
+     * 
+     */
+    
+    public function getSubpage(string $slug) : View|Response
+    {
+        $parent_page = LandingPageTemplate::getLanding('plan-your-visit');
+        $subpage = Subpages::getSubpage($slug);
+        $stubs_page = Stubs::getPage('plan-your-visit', $slug);
+
+        /**
+         * Maybe this block should be an if { } else { if {} else {} } instead
+         * In-case of a similar naming scheme for a newer, but unrelated page
+         */
+
+        // First, check the subpages collection
+        if(!empty($subpage['data'] && empty($stubs_page['data']))) {
+            return view('support.subpage', [
+                'page' => Collect($subpage['data'])->first(),
+                'parent_page' => Collect($parent_page['data'])->first()
+            ]);
+        }
+        // If that fails, then check the pages collection 
+        elseif(empty($subpage['data'] && !empty($stubs_page['data']))) {
```

Comment:
I think it would be better for this logic to work like this:

1. Does the page exist in the new collection?
2. If not, does it exist in the old collection?
3. if not, throw a 404 error page

At present if the page exists within both collections under the same encoded name, then it will result in a 404 error, which is misleading.

We should be pushing the client to use the new collections, so I think defaulting to an item from the new collections (if one exists) is sensible